jetty-util/src/main/java/org/eclipse/jetty/util/Pool.java

```diff
@@ -176,14 +231,15 @@ public Entry reserve(int allotment)
      * @param idx the index of the entry to acquire.
      * @return the specified entry or null if there is none at the specified index or if it is not available.
      */
+    @Deprecated
```

Comment:
Add `@deprecated` javadoc too.

Reply:
I guess we can't just remove it... will do in 10